index.js

```diff
@@ -1,21 +1,86 @@
-document.getElementById('open-bar').onclick = function() {
```

Comment:
Your image            |  Original image
:-------------------------:|:-------------------------:
<img src="https://user-images.githubusercontent.com/69011963/210364007-7b6cf919-b144-490d-b495-baaa4996911c.png" width="425"/> | <img src="https://user-images.githubusercontent.com/69011963/210363933-243903d6-4951-45e4-805e-2630703c2add.png" width="425"/> 

You did a great job however your design makes some differences compared to the original design. Kindly fix them step by step.

- [ ]   1. Increase the size of the title. 📐
- [ ]   2. Reduce the size of the close button. ❎
- [ ]   3. Expand the width of the content in relation to the original. 👈
- [ ]   4. Ensure that the See Live and See Source buttons are aligned and sized appropriately. 
- [ ]   5. Utilize any remaining space by filling it inappropriately. 🔭
- [ ]   6. Increase font size for improved legibility. 🔦